src/pages/LandingPage.tsx

```diff
@@ -47,76 +50,66 @@ export const LandingPage: FunctionComponent = () => {
-            </Form>
-          </SidebarPanel>
-          <SidebarContent className="pf-u-display-flex pf-u-flex-direction-column">
+  return <>
+    <Helmet>
+      <title>API Docs</title>
+    </Helmet>
+    <Page className="apid-c-page-landingpage pf-u-background-color-100">
+      <Sidebar>
+        <SidebarPanel className="pf-u-p-lg">
+          <Form>
+            <SearchInput
+              placeholder="Find by product or service name"
+              value={searchInput}
+              onChange={(_event, searchInput) => onChange(searchInput)}
+              onClear={() => onChange('')}
+            />
+            <SidebarTags tags={apiLabels} selected={selectedTags} setSelected={setSelectedTags} />
+          </Form>
+        </SidebarPanel>
+        <SidebarContent className="pf-u-display-flex pf-u-flex-direction-column">
 
-            <PageSection variant={PageSectionVariants.darker} className="pf-u-px-2xl-on-md pf-u-pb-2xl pf-u-background-color-dark-100">
-              <TextContent>
-                <Text component={TextVariants.h1}>The Red Hat API Documentation and Guides</Text>
-                <Text component={TextVariants.p}>
-                  Here you'll find APIs for many Red Hat products and services.
-                  Check back regularly as we're adding new ones all the time.
-                </Text>
-              </TextContent>
-            </PageSection>
+          <PageSection variant={PageSectionVariants.darker} className="pf-u-px-2xl-on-md pf-u-pb-2xl pf-u-background-color-dark-100">
+            <TextContent>
+              <Text component={TextVariants.h1}>The Red Hat API Documentation and Guides</Text>
+              <Text component={TextVariants.p}>
+                Here you'll find APIs for many Red Hat products and services.
+                Check back regularly as we're adding new ones all the time.
+              </Text>
+            </TextContent>
+          </PageSection>
 
-            <PageSection variant={PageSectionVariants.light} className="pf-u-p-md">
-              <div className="pf-u-text-align-right">
-                <Button variant="link" icon={<ThIcon />} className="pf-u-mr-sm" isInline isLarge/>
-                <Button variant="link" icon={<ThListIcon />} isInline isLarge isDisabled/>
-              </div>
-            </PageSection>
+          <PageSection variant={PageSectionVariants.light} className="pf-u-p-md">
+            <div className="pf-u-text-align-right">
+              <Button isDisabled={view === 'grid'} variant="link" icon={<ThIcon />} onClick={() => setView('grid')} className="pf-u-mr-sm" isInline isLarge/>
+              <Button isDisabled={view === 'list'} variant="link" icon={<ThListIcon />} onClick={() => setView('list')} isInline isLarge/>
+            </div>
+          </PageSection>
 
-            <PageSection className="apid-c-page__main-section-gallery" style={galleryPageStyle} padding={{ default: 'noPadding' }} isFilled={true}>
```

Comment:
This `style` should stay for the gallery page - looks like it got overwritten when rebasing/fixing conflicts